src/app/data-planes/routes.ts

```diff
@@ -57,6 +57,32 @@ export const routes = () => {
 
                 ],
               },
+              {
+                path: 'builtin-gateway-inbound/:service',
+                name: 'data-plane-builtin-gateway-inbound-summary-view',
+                component: () => import('@/app/data-planes/views/DataPlaneBuiltinGatewaySummaryView.vue'),
+                children: [
+                  {
+                    path: 'overview',
+                    name: 'data-plane-builtin-gateway-inbound-summary-overview-view',
+                    component: () => import('@/app/data-planes/views/DataPlaneBuiltinGatewaySummaryOverviewView.vue'),
+                  },
+                ],
+              },
+              {
+                path: 'builtin-gateway-outbound/:service',
+                name: 'data-plane-builtin-gateway-outbound-summary-view',
+                // TODO: Create dedicated outbound component
+                component: () => import('@/app/data-planes/views/DataPlaneBuiltinGatewaySummaryView.vue'),
+                children: [
+                  {
+                    path: 'overview',
+                    name: 'data-plane-builtin-gateway-outbound-summary-overview-view',
+                    // TODO: Create dedicated outbound component
+                    component: () => import('@/app/data-planes/views/DataPlaneBuiltinGatewaySummaryOverviewView.vue'),
+                  },
+                ],
+              },
```

Comment:
How come we need different Views for gateways, can't we just use the DataPlane ones that already exist?

Reply:
I’m not totally sure about this, yet. Simply put: I don’t know _if_ there are more differences or less. One difference so far is that for standard proxies, we show "Inbound :8080", but that doesn’t work for gateways. That’s why I split them up for now.

That being said, this might totally not how things will shake out. I’ve not made any changes and mostly just copied the summary view content into them.

Reply:
Ok cool, well my opinion on the general direction here would be:

As things stand they should probably just reuse the same views, similar to how the detail views and tabs are reused between sidecars and dataplanes currently.

Plus there are still things I want to tighten up here and I wouldn't want to duplicate too much _if_ I could avoid it

But will leave it up to you for the min